examples/twittermap/web/public/javascripts/map/countmap.js

```diff
@@ -183,17 +124,33 @@ angular.module('cloudberry.map')
             resultSet[0] = [];
             cloudberry.commonTimeSeriesResult = TimeSeriesCache.getValuesFromResult(cloudberry.timeSeriesQueryResult).concat(
               TimeSeriesCache.getTimeSeriesValues(cloudberry.parameters.geoIds, cloudberry.parameters.geoLevel, requestTimeRange));
-            cloudberry.countmapMapResult = resultSet[1].concat(cloudberry.countmapPartialMapResult);
+            if (cloudberry.parameters.geoLevel === "city") {
+              var newCities = [];
+              for (i = 0; i < cloudberry.parameters.geoIds.length; i++) {
+                newCities.push(cloudberry.parameters.geoIds[i]);
+              }
+              PopulationCache.loadCityPopulationToCache(newCities).done(function(){
```

Comment:
There may be a problem here. For this cache not all hit case, the callback function of `cloudberryClient.send` could be called multiple times if `slice` option is `ON`, then this `loadCityPopulationToCache` function could be called multiple times also, which might not be necessary.